specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2016-12-01/bms.json

```diff
@@ -2946,6 +2969,10 @@
           "description": "Indicates if protectable item is auto-protectable",
           "type": "boolean"
         },
+        "isAutoProtected": {
```

Comment:
A Boolean will forever have two valid values (true or false). A more flexible and future proof option is to define a property as a string with an enum having values true and false. This allows additional values to be added in the future in a non-breaking way. 

Reply:
We need boolean here as it conveys meaning strongly than enum and there is no plan for adding more values.

Reply:
It looks like this type of flag is used all over your resources with the redundant convention of the 'is' prefix. To retain consistency with the rest of your model, this is ok.

Reply:
Thanks for the feedback. We'll definitely consider it next time we want to release a breaking change.